api/bases/test.openstack.org_tempests.yaml

```
@@ -134,8 +132,7 @@ spec:
                 type: boolean
               storageClass:
                 default: local-storage
-                description: Name of a storage class that is used to create PVCs for
-                  logs storage. Required if default storage class does not exist.
+                description: StorageClass used to create PVCs that store the logs
```

Since we are in the mode of polishing stuff let's generalize it a little bit.
```suggestion
                description: StorageClass used to create any test-operator related PVCs.
```